src/application/BlogEditorPage/components/Form/components/FieldSet/FieldSet.tsx

```diff
@@ -27,6 +27,11 @@ const FieldSet: FC<Props> = ({
 }) => {
   const { register } = useFormContext()
 
+  useEffect(() => {
```

Comment:
The requirement is the style problem. It is better to change style instead of using effect.